x/incentives/keeper/abci_test.go

```diff
@@ -0,0 +1,58 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	distributionTypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	"github.com/peggyjv/sommelier/v4/app/params"
+	incentivesTypes "github.com/peggyjv/sommelier/v4/x/incentives/types"
+)
+
+func (suite *KeeperTestSuite) TestEndBlockerIncentivesDisabledDoesNothing() {
+	ctx, incentivesKeeper := suite.ctx, suite.incentivesKeeper
+	require := suite.Require()
+
+	incentivesParams := incentivesTypes.DefaultParams()
+	incentivesKeeper.SetParams(ctx, incentivesParams)
+
+	// By not mocking any other calls, the test will panic and fail if an unmocked keeper function is called,
+	// implying that the function isn't exiting early as designed.
```

Comment:
I commented this but in some scenarios it seemed like it would not panic when I expected it to (a mock call was not hit but no panic). Not sure if we want to investigate rn or not.

Reply:
I can investigate during testing.